templates/default/tez-site.xml.erb

```diff
@@ -38,6 +38,16 @@
     <type>integer</type>
   </property>
 
+  <!-- Number of tasks in mapper --> 
+  <property>
+    <name>tez.grouping.min-size</name>
+    <value>16000000</value>
+  </property>
+  <property>
+    <name>tez.grouping.max-size</name>
```

Comment:
This controls the size of task in mapper. Setting a smaller value will increase number of tasks of mapper.